src/vs/editor/contrib/codeAction/browser/codeActionModel.ts

```diff
@@ -335,6 +336,8 @@ export class CodeActionModel extends Disposable {
 				if (!isManualToAutoTransition) {
 					this.setState(newState);
 				}
+				const toDisposeActions = await actions;
+				toDisposeActions.dispose();
```

Comment:
code action goes off, but we get this notification: 
<img width="453" alt="Screenshot 2024-04-01 at 3 20 34 PM" src="https://github.com/microsoft/vscode/assets/54879025/0e2b011c-3ed4-498e-a7f9-f577a19d769a">

get this when not using nearby code actions (actually triggering on the diagnostic). likely because this calls and returns `getCodeActions` directly.